edx_adapt/tests/taurus_load_tests.yml

```diff
@@ -1,67 +1,125 @@
+        extract-regexp:
+          status_code:
+            regexp: (.+)
+            subject: http-code
+      - if: '"${status_code}" == "404"'
+        then:
+          # Configure skills
+          - url: /api/v1/parameters/bulk
+            method: POST
+            headers:
+              Content-Type: application/json
+            body:
+              course_id: "Adapt+A001+2017:7fc07e783e3645508eb6e996300a5e63"
+              params: {"threshold": 0.99, "ps": 0.25, "pi": 0.1, "pg": 0.25, "pt": 0.5}
+              user_id: "${anonym_id}"
+              skills_list: ["None"]
+          # Register user
+          - url: /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user
+            method: POST
+            headers:
+              Content-Type: application/json
+            body:
+              user_id: "${anonym_id}"
+  work_with_problem:
+    default-address: https://edx-adapt-dev.raccoongang.com
+    requests:
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
+      - url: /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}/pageload
         method: OPTIONS
-      - http://52.210.53.36:8080/api/v1/course/CMUSTAT101/user/student21
-      - url: http://52.210.53.36:8080/api/v1/course/CMUSTAT101/user/student21/pageload
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
+      - url: /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}/pageload
         method: POST
         headers:
           Content-Type: application/json
-        body: '{"problem": "T3"}'  # post JSON
-      - http://52.210.53.36:8080/api/v1/course/CMUSTAT101/user/student21
-      - http://52.210.53.36:8080/api/v1/course/CMUSTAT101/user/student21
+        body:
+          problem: "Post_test_0"
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
+      - /api/v1/course/Raccoon+RG001+2017:pd_sampling_training/user/${anonym_id}
 
+  main_test_scenario:
+    data-sources:
+      - path: ../../data/taurus_test_users.csv
+        loop: true
+    think-time: 50ms
+    retrieve-resources: true
+    store-cookie: true
+    timeout: 10s
+    requests:
+      - include-scenario: login-and-open-intro-page
+      - include-scenario: xblock-enrolling
+      - include-scenario: work_with_problem
 
 reporting:
-- module: final_stats
-  dump-csv: taurus_load_tests_report.csv
-- module: console
+  - module: final_stats
+    dump-csv: taurus_adapt_load_tests_report_100-local.csv
+  - module: console
+  - module: blazemeter
+    report-name: Local 100 course RG
+    test: only Adapt Load Test
+    project: 158825
+
+modules:
+  jmeter:
+    path: ~/edx-adapt/apache-jmeter-3.1
+
+  cloud:
```

Comment:
I've tried it but free account allow only 50 concurrent users so it is not really interesting